### PR TITLE
Combined prefix operations

### PR DESCRIPTION
**Pull request by rileymckenna**
Added peering service events
Changed the naming of enum types
Added events to peering service prefixes

[SPEC PR](https://github.com/Azure/azure-rest-api-specs/pull/7363)

### CODE REVIEW

File: sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/PrefixesOperationsExtensions.cs [hunk 1]
```diff
@@ -18,47 +18,209 @@ namespace Microsoft.Azure.Management.Peering
...
+            /// <param name='prefixName'>
+            /// The name of the prefix.
+            /// </param>
+            /// <param name='peeringServicePrefix'>
+            /// The properties needed to create a prefix.
+            /// </param>
+            public static PeeringServicePrefix CreateOrUpdate(this IPrefixesOperations operations, string resourceGroupName, string peeringServiceName, string prefixName, PeeringServicePrefix peeringServicePrefix)
             {
-                return operations.ListByPeeringServiceAsync(resourceGroupName, peeringServiceName).GetAwaiter().GetResult();
+                return operations.CreateOrUpdateAsync(resourceGroupName, peeringServiceName, prefixName, peeringServicePrefix).GetAwaiter().GetResult();
```
Comment (AlexanderSher) [inline sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/PrefixesOperationsExtensions.cs]: Don't we have to pass `CancellationToken` to `CreateOrUpdateAsync`?

Comment (rileymckenna) [reply sdk/peering/Microsoft.Azure.Management.Peering/src/Generated/PrefixesOperationsExtensions.cs]: @AlexanderSher i am not sure? this is auto generated code.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # Service teams
 /sdk/appconfiguration/ @annelo-msft
```
Comment (markcowl) [inline .github/CODEOWNERS]: Please include the generation metadata file update (output from generate.ps1) here: `eng\mgmt\mgmtmetadata\<rpName>.txt`

**Review by markcowl — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (rileymckenna) [reply .github/CODEOWNERS]: @markcowl @AlexanderSher i added the meta data file into the project.

**Review by markcowl — APPROVED** [review]

**Pull request merged**